packages/common/styles/src/lib/notional-theme/notional-theme.ts

```diff
@@ -353,7 +353,7 @@ function getTypography(themeVariant: PaletteMode, pageLayout?: NotionalPageLayou
         marginBottom: pxToMargin(12),
       },
       link: {
-        fontSize: pxToRem(12),
+        fontSize: pxToRem(14),
```

Comment:
Is this an "approved" change to the typography?

Reply:
Approved by me lol. I checked and we don't use that "link" anywhere. In typography it's only used by LinkText. Which is not used anywhere but my new component.